megamek/src/megamek/client/ui/dialogs/BVDisplayDialog.java

```diff
@@ -42,16 +42,10 @@ public BVDisplayDialog(final JFrame frame, final boolean modal, final Entity ent
     }
     //endregion Constructors
 
-    //region Getters
-    public Entity getEntity() {
-        return entity;
-    }
-    //endregion Getters
-
     //region Initialization
     @Override
     protected Container createCenterPane() {
-        final JEditorPane editorPane = new JEditorPane("text/html", getEntity().getBVText());
+        final JEditorPane editorPane = new JEditorPane("text/html", entity.getBVText());
         editorPane.setEditable(false);

```

Comment:
Revert. This is a stylistic decision that I tend to use for code I work with, namely using getters/setters instead of the variable itself.